index.html

```diff
@@ -85,10 +87,14 @@ <h2 class="title">
           <span class="title-3-gradient background-clip-text">공유</span>해 보세요
         </h2>
         <p class="description">
-          여러 링크를 폴더에 담고 공유할 수 있습니다.
-          <br />
-          가족, 친구, 동료들에게 쉽고 빠르게 링크를
-          <br />
+          여러 링크를 폴더에 담고 공유할 수 
+          <br class="description-br-tablet" />
+          있습니다.
+          <br class="description-br-pc"/>
+          가족, 친구, 동료들에게 쉽고 
+          <br class="description-br-tablet" />
+          빠르게 링크를
+          <br class="description-br-pc"/>
```

Comment:
너비에 따라 줄바꿈되는 글자가 바뀌는 군요. 마크업 구조나, 줄바꿈 방식을 바꿔서 더 단순해질수도 있을거같아요. 협의하에 어느정도는 컨테이너크기에 맞게 줄바꿈되게 하거나, word-break 속성을 줄수도 있겠죠. 이런 부분은 실제 프로젝트에서는 유연하게 결정하셔서 코드도 자연스러워지면 좋겟네요

Reply:
```suggestion
          여러 링크를 폴더에 담고 공유할 수 
          <br class="hidden-tablet" />
          있습니다.
          <br class="hidden-pc"/>
          가족, 친구, 동료들에게 쉽고 
          <br class="hidden-tablet" />
          빠르게 링크를
          <br class="hidden-pc"/>
```

클래스이름만 제안드리자면, 위와 같이 유틸리티 성격의 클래스를 선언해두시고 쓰면 유용할거같습니다.

[비슷한 맥락에서 드린 코멘트](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/pull/72#discussion_r1382509487)도 있어서 같이 보시길 추천드립니다.